web/src/compose_banner.ts

```diff
@@ -119,7 +119,7 @@ export function clear_all(): void {
 }
 
 export function show_error_message(
-    message: string,
+    message: string | null | undefined,
```

Comment:
This change seems to be not commented on in the commit message, and I don't see why it would be necessary or correct as part of this change -- going to revert this hunk and see if CI passes after doing so.